src/main/java/mods/railcraft/common/gui/slots/SlotPassThrough.java

```diff
@@ -18,11 +18,6 @@ public SlotPassThrough(IInventory inv, int slotIndex, int posX, int posY) {
         super(inv, slotIndex, posX, posY);
     }
 
-    @Override
-    public int getSlotStackLimit() {
```

Comment:
Why is this removed? This specific Slot should mirror the underlying inventory.

Reply:
That is set to a field in the basic slot class.

Reply:
Yes, but this specific slot is different. It is designed to simply look at the underlying inventory and get the info from there.

Reply:
Hmm. Involves some design thing. Will look into it more.